### ISSUE TITLE
Somehow I keep accidentally archiving text conversations and then unarchiving them 

### ISSUE THREAD

**Issue by successtheman** [jerrod-lankford/google-voice-desktop-app#141]
I am not sure how it is happening but is there some keyboard shortcut I can disable or something? It is very annoying and happens whenever I am typing text messages

**Comment by dave15559546** [comment]
same thing happens to me though less as of recent

I suspect it's a function of chrome updates

**Comment by klepper** [comment]
Yes, this is so bad. Accidentally start typing not in the box? Conversation was archived. DISABLE THESE ASAP.

**Comment by successtheman** [comment]
It seems to have stopped as of lately for me but I could just not be pressing the same keys I was when it happened before. Unfortunately I don't know what those keys were otherwise I would test it.
